packages/contentful--app-scripts/lib/utills.test.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+
+const { isValidIpAddress, removeProtocolFromUrl } = require('./utils');
+
+describe('isValidIpAddress', () => {
+  it('returns true for a valid IP address', () => {
+    const result = isValidIpAddress('192.168.0.1');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns true for a valid domain', () => {
+    const result = isValidIpAddress('google.com');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns true for a valid ipv6 address', () => {
+    const result = isValidIpAddress('2001:0db8:85a3:0000:0000:8a2e:0370:7334');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns true for a valid domain with port', () => {
+    const result = isValidIpAddress('google.com:4000');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns true for a valid IP address with port', () => {
+    const result = isValidIpAddress('192.168.0.1:2000');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns false for an invalid IP address', () => {
+    const result = isValidIpAddress('not an ip address');
+    assert.strictEqual(result, false);
+  });
+
+  it('returns true for an valid ipv6 address with port', () => {
+    const result = isValidIpAddress('[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:443');
+    assert.strictEqual(result, true);
+  });
+});
+
+describe('removeProtocolFromUrl', () => {
+  it('returns the host of a URL without a protocol', () => {
+    const result = removeProtocolFromUrl('http://example.com');
+    assert.strictEqual(result, 'example.com');
+  });
+
+  it('returns valid ipv6 address', () => {
+    const result = removeProtocolFromUrl('http://[2001:0db8:85a3:0000:0000:8a2e:0370]:7334');
+    assert.strictEqual(result, '[2001:0db8:85a3:0000:0000:8a2e:0370]:7334');
+  });
+
+  it('returns valid domain', () => {
+    const result = removeProtocolFromUrl('example.com');
```

Comment:
Should have protocol here, right? Or we can test both cases – with and without a protocol.

Reply:
Ah, nevermind, you test it above.